Navigation/Signoffs/Livestream Camera Subsystem.md

```diff
@@ -0,0 +1,97 @@
+# Livestream Camera Subsystem
+# Function of the subsystem
+
+Adding a live stream camera to the Autonomous Crawlspace Inspection Robot allows for real-time visual monitoring of the crawlspace. This helps identify issues, make informed decisions, and ensure a smooth viewing experience with clear video transmission and a stable connection.
+
+
+# Constraints:
+
+| Item:          | C02 Camera              |
+|----------------|-------------------------|
+| Image Sensor:  | 1/4" CMOS Sensor        |
+| Horizontal:    | 1200TVL with Global WDR |
+| Signal System: | NTSC                    |
+| Lens:          | 2.1mm(M8) FOV 160°      |
+| S/N Ratio:     | >50db                   |
+| Power:         | DC 3-5.5V               |
+| Current:       | 110mA@5V / 120mA@3.3V   |
+| Dimension:     | 13*11mm                 |
+| Net weight:    | 1.4g                    |
+
+
+The first constraint is the bandwidth required for streaming video. It depends on resolution, frame rate, and compression. The camera provides a video stream with a resolution of 1200 TVL (1280x1024) at 30 fps, using H.264 compression (compression ratio of 20:1 for H.264). The required bandwidth is calculated to be 78.64 Mbps. Sufficient bandwidth is essential for seamless video streaming. When the available bandwidth is lower than the required minimum, it can lead to buffering or reduced video quality. The network conditions are unstable or restricted, consistently meeting the bandwidth requirement becomes difficult and impacts the viewing.
```

Comment:
Why is there a table above that does not contain constraints but there are constraints here in paragraph form?

Please number the constraints and give them in a table or list. 

Your analysis should address the numbered constraints in exactly the same order.